cfpb/cfgov-refresh
# Enable the sortable tables scripting

Now that Capital Framework has been updated in this project, the scripting for sortable tables is available. Added an on-demand script that initializes the scripting when tables are used on a page.

## Additions

- Added on-demand script to initialize cf-tables code when a page includes a table

## Testing

1. Locally, turn on the sortable tables flag
<img width="758" alt="screen shot 2017-08-10 at 10 36 09 am" src="https://user-images.githubusercontent.com/1280430/29178634-bf9f2e72-7db7-11e7-8e33-90444a43caed.png">

2. Create a page with a table inside a full-width text module, enable sortable table functionality and set the sorting options.

<img width="644" alt="screen shot 2017-08-10 at 10 37 00 am" src="https://user-images.githubusercontent.com/1280430/29178670-e2809df4-7db7-11e7-96a9-20b00faa16c4.png">

3. Save page and view it in the browser

<img width="704" alt="screen shot 2017-08-10 at 10 37 39 am" src="https://user-images.githubusercontent.com/1280430/29178695-f28d7a6e-7db7-11e7-9fcd-4f2a26c13de4.png">

4. Click the headings and the table should resort itself.

## Notes

- There is a bug in the chart builder override file causing the heading colors to be incorrectly set making it dark on dark. This needs to be fixed separately, but because the sortable functionality is behind a flag, it will not be an immediate issue.

## Todos

- Update CF to use real variables to avoid override bugs

## Checklist
